2/scss/media-tablet.scss

```diff
@@ -0,0 +1,217 @@
+@import "vars";
+@import "mixins";
+
+@media (min-width: 768px) {
```

Comment:
please, write main styles and media-queries right inside same block.

Example:
```
.header {
  padding: 16px 0;

  @media (min-width: 768px) {
    padding: 32px 0;
  }
}
```

Reply:
transferred styles media-queries